### DISCUSSION TITLE
Error Installing Uncover

### DISCUSSION THREAD

**Discussion by pseudog0d** [projectdiscovery/uncover#204] [Q&A]
### uncover version:
NA

### Current Behavior:
While trying to install Uncover I am getting the following error:

![image](https://user-images.githubusercontent.com/65175780/228646377-ff1affd8-7579-4fba-b7d5-709929ac4427.png)

Error:

github.com/modern-go/reflect2
gopkg.in/yaml.v3
# github.com/modern-go/reflect2
go/pkg/mod/github.com/modern-go/reflect2@v1.0.2/unsafe_map.go:9:2: undefined: unsafeType
go/pkg/mod/github.com/modern-go/reflect2@v1.0.2/unsafe_link.go:18:55: undefined: sliceHeader
go/pkg/mod/github.com/modern-go/reflect2@v1.0.2/go_above_118.go:17:20: type2.rtype undefined (type *UnsafeMapType has no field or method rtype)
go/pkg/mod/github.com/modern-go/reflect2@v1.0.2/reflect2.go:145:22: undefined: unpackEFace
go/pkg/mod/github.com/modern-go/reflect2@v1.0.2/reflect2.go:157:22: undefined: unpackEFace
go/pkg/mod/github.com/modern-go/reflect2@v1.0.2/reflect2.go:168:14: undefined: safeType
go/pkg/mod/github.com/modern-go/reflect2@v1.0.2/reflect2.go:172:12: undefined: safeStructType
go/pkg/mod/github.com/modern-go/reflect2@v1.0.2/reflect2.go:174:10: undefined: newUnsafeStructType
go/pkg/mod/github.com/modern-go/reflect2@v1.0.2/reflect2.go:177:12: undefined: safeSliceType
go/pkg/mod/github.com/modern-go/reflect2@v1.0.2/reflect2.go:179:10: undefined: newUnsafeArrayType
go/pkg/mod/github.com/modern-go/reflect2@v1.0.2/reflect2.go:179:10: too many errors
# gopkg.in/yaml.v3
go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/decode.go:33:11: undefined: yaml_parser_t
go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/decode.go:34:11: undefined: yaml_event_t
go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/decode.go:35:12: undefined: Node
go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/decode.go:36:23: undefined: Node
go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/decode.go:80:27: undefined: yaml_event_type_t
go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/decode.go:99:25: undefined: yaml_event_type_t
go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/decode.go:140:28: undefined: Node
go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/decode.go:147:27: undefined: Node
go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/decode.go:314:11: undefined: Node
go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/decode.go:315:15: undefined: Node
go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/decode.go:147:27: too many errors

OS Info:
NAME=EndeavourOS
PRETTY_NAME=EndeavourOS
ID=endeavouros
ID_LIKE=arch

**Comment by ehsandeep** [comment]
@pseudog0d you need to have go version installed  => 1.19 to install uncover from the source or you can install it using https://github.com/projectdiscovery/pdtm
